File: firstAPP/LZHFirstAPP/LZHFirstAPP/LZHViewController.m
//
//  LZHViewController.m
//  LZHFirstAPP
//
//  Created by LZHmac pro on 16/3/9.
//  Copyright © 2016年 李志宏. All rights reserved.
//

#import "LZHViewController.h"

@interface LZHViewController ()
{   NSArray *questions;
    NSArray *answers;
    int currQuestion;
}
@end

@implementation LZHViewController

- (IBAction)nextQuestions:(id)sender {
    ++currQuestion;
    
    
    if (currQuestion == 3) {
        currQuestion = 0;
    }
    _questionsLable.text = questions[currQuestion];
}
- (IBAction)showQuestions:(id)sender {
   
    if (currQuestion == 3) {
        currQuestion = 0;
    }
     _answerLable.text = answers[currQuestion];
}

- (void)viewDidLoad {
    [super viewDidLoad];
    questions = @[@"1what is your name",@"2who are you?",@"3where are you from"];
    answers = @[@"1LZH",@"2lizhihong",@"3anhui"];
    currQuestion = 0;
    _questionsLable.text = questions[currQuestion];
    _answerLable.text = @"???";
    
    // Do any additional setup after loading the view.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

/*
#pragma mark - Navigation

// In a storyboard-based application, you will often want to do a little preparation before navigation
- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {
    // Get the new view controller using [segue destinationViewController].
    // Pass the selected object to the new view controller.
}
*/

@end

File: firstAPP/FirstAPP--swift/FirstAPP--swift/LZHViewController.swift
//
//  LZHViewController.swift
//  FirstAPP--swift
//
//  Created by LZHmac pro on 16/3/9.
//  Copyright © 2016年 李志宏. All rights reserved.
//

import UIKit

class LZHViewController: UIViewController {
    
    @IBOutlet weak var question: UILabel!

    @IBOutlet weak var answer: UILabel!
    
    @IBAction func ShowQuestions(sender: AnyObject) {
        ++curr
        //＝3的时候会溢出 所以当＝3的时候让它归零
        if(curr == 3){curr = 0}
        question.text = questions[curr]
        //++curr // .count  是数组自带的方法  用来检索当前索引 是一个
        //if curr == questions.count{curr = 0}
    }
    
    @IBAction func ShowAnswer(sender: AnyObject) {
        answer.text = answers[curr]
        if(curr == 3){curr = 0}
        //answer.text = answers[curr]
    }
    var questions = ["question1","question2","question3"]
    
    var answers = ["answer1","answer2","answer3"]
    
    var curr = 0
    
    
    override func viewDidLoad() {
        super.viewDidLoad()
        question.text = questions[curr];
        answer.text = "???"
        // Do any additional setup after loading the view.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

    /*
    // MARK: - Navigation

    // In a storyboard-based application, you will often want to do a little preparation before navigation
    override func prepareForSegue(segue: UIStoryboardSegue, sender: AnyObject?) {
        // Get the new view controller using segue.destinationViewController.
        // Pass the selected object to the new view controller.
    }
    */

}
